### ISSUE TITLE
ELRS PWM firmware mod

### ISSUE THREAD

**Issue by wimalopaan** [frank26080115/EdgeTX-Scripts-and-FW-Collection#1]
Please make your modified firmware source public. (E.g. make a fork of elrs so that other can use your modifications).

**Comment by frank26080115** [comment]
it's here https://github.com/frank26080115/ExpressLRS/tree/3.3.1-nopulse

the servo code got a insanely major upstream refactoring right after I made the changes so it's going to be hard to continue developing that branch and also hard to pull request my changes

**Comment by wimalopaan** [comment]
If I look at your commits, it looks like the changes to simpy produce pwm-output are in the .json file? The other changes apply to the nopulse-changes you made.

**Comment by frank26080115** [comment]
yes your understanding is correct, PWM only needs a reconfiguration via JSON or web interface, the nopulse failsafe required an actual firmware change to javascript and C/C++ code

**Comment by wimalopaan** [comment]
Ok, thanks!!!

**Comment by frank26080115** [comment]
I did a cleaner pull request, it's different but it's against their latest master https://github.com/ExpressLRS/ExpressLRS/pull/2535
